schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+                      - required:
+                          - required
+                        properties:
+                          required:
+                            const: true
+                            default: true
+                  - allOf:
+                      - anyOf:
+                          - $ref: '#QueryParam'
+                          - $ref: '#HeaderParam'
+                          - $ref: '#CookieParam'
+                      - $ref: '#/$defs/Mixins/$defs/DescReqDep'
+            oneOf:
+              - allOf:
+                  - required:
+                      - schema
+                  - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+                  - oneOf:
+                      - allOf:
+                          - $ref: '#PathParam'
+                          - $ref: '#/$defs/Mixins/$defs/StyledMatrix'
+                      - allOf:
+                          - $ref: '#QueryParam'
+                          - $ref: '#/$defs/Mixins/$defs/StyledFormComplex'
+                      - allOf:
+                          - $ref: '#HeaderParam'
+                          - $ref: '#/$defs/Mixins/$defs/StyledSimple'
+                      - allOf:
+                          - $ref: '#CookieParam'
+                          - $ref: '#/$defs/Mixins/$defs/StyledFormOnly'
+              - allOf:
+                  - required:
+                      - content
+                  - $ref: '#/$defs/Mixins/$defs/WithContent'
+                  - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+            unevaluatedProperties: false
+        $defs:
+          PathParam:
+            $anchor: PathParam
+            properties:
+              in:
+                const: path
+          QueryParam:
+            $anchor: QueryParam
+            properties:
+              in:
+                const: query
+          HeaderParam:
+            $anchor: HeaderParam
+            properties:
+              in:
+                const: header
+          CookieParam:
+            $anchor: CookieParam
+            properties:
+              in:
+                const: cookie
+      RequestBody:
+        $anchor: RequestBody
+        required:
+          - content
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Requireable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          content:
+            type: object
+            additionalProperties:
+              $ref: '#MediaType'
+        unevaluatedProperties: false
+      SecurityScheme:
+        $anchor: SecurityScheme
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - type
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              type:
+                string: null
```

Is this maybe suposed to be
```suggestion
                type: string
```
?